src/FindARoommate/FindARoommate.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useEffect } from 'react';
+import getAllResidents from '../apiCalls';
+import ControlledOpenSelect from '../Form/Form';
+import SingleLineGridList from '../Thumbnails/Thumbnails'
+
+export default function FindARoomate() {
+  const [semester, setSemester] = useState('');
+  const [residents, setResidents] = useState([]);
+
+  useEffect(() => {
+    getResidents();
+  }, [])
+
+  const getResidents = async () => {
+    await getAllResidents()
+    .then(data => setResidents(data))
+    .catch(err => console.log(err))
+    console.log(residents)
+  }
+
+  const selectSemester = (selectedSemester) => {
+    setSemester(selectedSemester)
+  }
+  
+  return(
+    <div>
+      <ControlledOpenSelect selectSemester={selectSemester}/>
+      <SingleLineGridList availableResidents={residents} semesterAvailable={semester}/>
```

Comment:
We can change this prop (residents) to residentsAvailable once we're able to query residents on our server by the semester property